src/packages/next/pages/policies/thirdparties.tsx

```diff
@@ -34,293 +34,332 @@ export default function TermsOfService({ customize }) {
             <div style={{ fontSize: "12pt" }}>
               <p>
                 CoCalc uses a number of third party providers and services. For
-                each of them, we link to further policies, explain the purpose
-                and use case, and which data is shared.{" "}
+                each of them, we explain the purpose and use case, which
+                Personally Identifiable Information (PII) of our users is
+                shared, and give links to their privacy policies.{" "}
+              </p>
+              <h2>Essential Services</h2>
+              <p>
+                These services support CoCalc's infrastructure, you are likely
```

Comment:
This is a run-on sentence -- the comma should be a semicolon, or add a correlative conjunction like "so" or "and"...